scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -436,77 +413,47 @@ public void onClick(MenuItemClickEvent event) {
         submitButton.setIcon(SchedulerImages.instance.job_submit_16().getSafeUri().asString());
         submitButton.setIconSize(20);
         submitButton.setTooltip("Submit a new job");
-        submitButton.addClickHandler(new ClickHandler() {
-            public void onClick(ClickEvent event) {
-                SubmitWindow w = new SubmitWindow(SchedulerPage.this.controller);
-                w.show();
-            }
+        submitButton.addClickHandler(event -> {
+            SubmitWindow w = new SubmitWindow();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)